package.json

```diff
@@ -57,7 +57,10 @@
     "tsd": "^0.6.4",
     "typescript": "~1.6.0",
     "zone.js": "^0.5.8",
-    "yargs": "^3.25.0"
+    "yargs": "^3.25.0",
+    "typescript-node": "^0.1.3",
+    "typescript-register": "^1.1.0",
+    "typescript-require": "^0.2.9-1"
```

Comment:
Makes sense. Fixed that and removed `typescript-require`, I don't think it is required.

@briantopping would you test against this branch again?